mne_bids/config.py

```diff
@@ -289,7 +285,7 @@
 MNE_FRAME_TO_STR = {val: key for key, val in MNE_STR_TO_FRAME.items()}
 
 # see BIDS specification for description we copied over from each
-BIDS_COORD_FRAME_DESCRIPTIONS = {
+BIDS_COORD_FRAME_DESCRIPTIONS = dict(**{
```

Comment:
uh there's gotta be a better way to do this. In the worst case, simply first create a dict and then append to it / update it, please.